docs/how-tos/how_to_skip_rmas_in_returns.mdx

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 2
+---
+
+# How to skip RMAs in returns
+
+We'll see how you can disable RMAs as a prerequisite to creating a customer
+return. Solidus' [built-in process for returns][returns] is mighty and
+flexible. Only that sometimes a store doesn't need such a feature-rich system.
```

Comment:
Given that the robustness and flexibility of the returns system is exactly what allows you to enable/disable RMAs so easily, I don't know if the last two sentences have a strong logical connection.

How about reframing this in a more positive way?
```suggestion
Solidus' [built-in process for returns][returns] is mighty and flexible, and
even allows you to disable RMAs as a prerequisite to creating a customer
return.
```

Reply:
Yeah, it makes sense. Integrated.